ninetofiver/choices.py

```diff
@@ -0,0 +1,38 @@
+from ninetofiver.settings import REDMINE_URL, REDMINE_API_KEY
```

Comment:
Create a separate package for redmine-specific stuff. All redmine-related functionality should be completely optional.